### PR TITLE
Add note about pre-requests to contributing doc

### PR DESCRIPTION
**Pull request by kragniz**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,6 +7,18 @@ issue before making a change.  This saves everyone from wasted effort in the eve
 changes need some adjustment before they are ready for submission.
 All new code, including changes to existing code, should be tested and have a corresponding test added or updated where applicable.
 
+
+## Pre-requests
```
Comment (lw346) [inline CONTRIBUTING.md]:
```suggestion
## Prerequisites
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -7,6 +7,18 @@ issue before making a change.  This saves everyone from wasted effort in the eve
...
+
+The following must be installed on your development machine:
+
+- `docker`
+- `openjdk-8`
```
Comment (lw346) [inline CONTRIBUTING.md]: While installing `openjdk-8` will guarantee the requirement of a Java 8 JDK, we should probably make clear that other JDKs that fulfill the requirement are available.

Comment (kragniz) [reply CONTRIBUTING.md]: Sure, 
```
`openjdk-8` or another JDK
```
sound okay?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -7,6 +7,18 @@ issue before making a change.  This saves everyone from wasted effort in the eve
...
 
+
+## Prerequisites
+
+The following must be installed on your development machine:
+
+- `docker`
+- `openjdk-8` or another JDK
+- `gcc` (or `build-essential` package on debian distributions)
+- `kubectl`
```
Comment (sebbonnet) [inline CONTRIBUTING.md]: We could add `golang` and `dgoss` to this list too.

Comment (kragniz) [reply CONTRIBUTING.md]: Good idea.

Comment (kragniz) [reply CONTRIBUTING.md]: Though `make setup` installs `dgoss` in the "Building and Testing" step.

Comment (sebbonnet) [reply CONTRIBUTING.md]: It does you're right, but I don't think this is correct as it requires elevated privileges (`sudo`) to install `dgoss`. I have removed it in a follow up PR #64, so I think we should add the `dgoss` requirement here

Comment (kragniz) [reply CONTRIBUTING.md]: :+1:

**Pull request merged**